src/shim/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015"
```

Comment:
this babelrc is different from the top level project one. dunno if it is an oversight or it is intended....